### DISCUSSION TITLE
Suppression rapide d'image non-utilisées

### DISCUSSION THREAD

**Discussion by AmauryPit1** [witsa/synapps#787] [Ideas]
Bonjour, 
nous travaillons sur Synapps avec une banque d'image assez conséquente. 
Afin de réduire la taille du fichier .PRY et que le chargement du synapps sur REDY PC soit plus court, nous enlevons image par image celles qui sont inutilisées dans le programme synapps (manuellement…). Il serait confortable d'avoir un bouton qui permet de vérifier les images non utilisées et de les enlever.
Merci.

**Comment by Raphwit** [comment]
Hello, c'est une bonne idée, nous l'ajoutons à la liste des futurs ajouts à implémenter !

Cependant il est à noter qu'il est difficile de détecter l'utilisation d'une image mise en librairie, celle-ci peut se trouver dans une scène, un composite, un style, cela rendra la recherche peut être un peu couteuse mais nous nous pencherons sur le sujet, merci du retour !
